airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -114,6 +116,7 @@ def __init__(
         self.json_fields = [label.strip() for label in json_fields.split(",")]
         self.host_field = host_field
         self.offset_field = offset_field
+        self.index_patterns = index_patterns or es_configs.get('index_patterns', '_all')
```

Comment:
```suggestion
        self.index_patterns = index_patterns or conf.get('elasticsearch', 'index_patterns', fallback='_all')
```

Then you can drop 98 (`es_configs`). Also, can't we set this default on 86, letting us do a simple assignment here?